eng/code-quality-reports/pom.xml

```diff
@@ -31,6 +31,14 @@
   </scm>
 
   <dependencies>
+
+    <!-- Transient Direct Dependency until bump to Java 11 baseline -->
```

Comment:
Can you please add a comment to mention why we need to add the direct dependency?